mysqlchk.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Mysqlchk check
+After=mariadb.service
+
+[Service]
+ExecStart=/usr/bin/mysqlchk
```

Comment:
We should probably have a way to pass arguments to mysqlchk so people can override the default settings. How about using EnvironmentFile and passing something like $MYSQLCHK_ARGS on this line.

We should also drop privileges to nobody or something, there's no reason for this to run as root.